projects/dvl-fw/src/lib/plugins/nsf/nsf-template-project.ts

```diff
@@ -25,15 +27,17 @@ import { isNSFCompatibleCSV } from './nsf-validator';
 
 
 export class NSFTemplateProject extends DefaultProject {
-  static async create(nsfFileContent: string, fileName?: string): Promise<Project> {
+  static async create(nsfFileContents: string[] | string, fileNames?: string[] | string): Promise<Project> {
+    nsfFileContents = isArray(nsfFileContents) ? nsfFileContents : [nsfFileContents];
+    fileNames = isArray(fileNames) ? fileNames : [fileNames];
```

Comment:
Need to also take into account when fileNames is null...
`fileNames = isArray(fileNames) || !fileNames ? fileNames : [fileNames];`